language/Portuguese_Brazil/strings.po

```diff
@@ -200,35 +200,35 @@ msgstr "Adicionar filtro"
 
 #: src/frontend/mame/ui/utils.cpp:950
 msgid "Select category:"
-msgstr ""
+msgstr "Selecione a categoria"
 
 #: src/frontend/mame/ui/utils.cpp:981
 msgid "[no category INI files]"
-msgstr ""
+msgstr "[arquivos INI sem categorias]"
 
 #: src/frontend/mame/ui/utils.cpp:989
 msgid "[no groups in INI file]"
-msgstr ""
+msgstr "[sem grupos no arquivo INI]"
 
 #: src/frontend/mame/ui/utils.cpp:1023
 msgid "No category INI files found"
-msgstr ""
+msgstr "arquivos INI sem categoria encontrada"
```

Comment:
OK, the current text is probably not correct, but "INI files lacking category were found" doesn't mean the right thing.  It's saying that no INI files were found, specifically INI files that contain categories, not that it found INI files but they contain no categories.